src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1408,7 +1449,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 			});
 			this._storageService.store(SUGGESTED_RENDERER_TYPE, 'auto', StorageScope.GLOBAL, StorageTarget.MACHINE);
 		} catch (e) {
-			this._logService.warn(`Webgl could not be loaded. Falling back to the canvas renderer type.`, e);
+			this._logService.warn(`Webgl could not be loaded.Falling back to the canvas renderer type.`, e);
```

Comment:
This used to have a space at "...loaded.Falling back..." but its no longer there and is now incorrect spacing.